packages/callback-example/lib/__tests__/issuerCallback.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { KeyObject } from 'crypto'
+
+import { CredentialRequestClient, CredentialRequestClientBuilderV1_0_09, ProofOfPossessionBuilder } from '@sphereon/openid4vci-client'
+import {
+  Alg,
+  CredentialFormatEnum,
+  CredentialSupported,
+  Display,
+  IssuerCredentialSubjectDisplay,
+  Jwt,
+  ProofOfPossession,
+  Typ,
+} from '@sphereon/openid4vci-common'
+import { CredentialSupportedBuilderV1_11, VcIssuer, VcIssuerBuilder } from '@sphereon/openid4vci-issuer'
+import { MemoryCredentialOfferStateManager } from '@sphereon/openid4vci-issuer/dist/state-manager/MemoryCredentialOfferStateManager'
+import { ICredential, IProofPurpose, IProofType, W3CVerifiableCredential } from '@sphereon/ssi-types'
+import * as jose from 'jose'
+
+import { generateDid, getIssuerCallback, verifyCredential } from '../IssuerCallback'
+
+const INITIATION_TEST_URI =
```

Comment:
Why is the issuer using a v8/v9 URI, whilst the Issuer itself is v11+

Reply:
Fixed!

Reply:
That was not the point. The point was openid-initiate-issuance vs openid-credential-offer, so v8/9 vs v11.
But since the client does not fully support v11 for now, I guess it is okay for now